### DISCUSSION TITLE
trying to setup a DCCEX Command Station

### DISCUSSION THREAD

**Discussion by fwelto** [DCC-EX/CommandStation-EX#243] [General]
Hello,

I just setup a new DCCEX command station using an ELEGOO MEGA 2560 R3 and an Arduino Motor shield.  I did not setup the wifi option yet (as I want to make sure I have the basics working first).  I followed the tutorial here https://dcc-ex.com/get-started/assembly.html and uploaded the CommandStation sketch to the MEGA.

I  installed the DCCEX/exwebthrottle locally and ran it in Google Chrome.  I am able to connect to my MEGA on COM3.  Unfortunately, none of the commands seem to drive my loco.  No lights, horn, or motion to speak of.   Any ideas?  Let me know if I can provide more helpful details.

Note, I am able to send DCC commands directly from the Arduino IDE Serial Monitor - which work well.   I'm not sure why the exwebthreottle is not working tho.

Thanks!

Here's a copy of the debug console from the exwebthrottle.

```

[SEND]

[SEND]echo(false);

<

<

* License GPLv3 fsf.org (c) dcc-ex.com *> <* LCD0:DCC++ EX v4.0.0 *> <* LCD1:Lic

* License GPLv3 fsf.org (c) dcc-ex.com *> <* LCD0:DCC++ EX v4.0.0 *> <* LCD1:Lic

GPLv3 *> <* MotorDriver currentPin=A1, senseOffset=0, rawCurrentTripValue(relative to offset)=66

GPLv3 *> <* MotorDriver currentPin=A1, senseOffset=0, rawCurrentTripValue(relative to offset)=66

8 *> <* MotorDriver currentPin=A0, senseOffset=0, rawCurrentTripValue(relativ

8 *> <* MotorDriver currentPin=A0, senseOffset=0, rawCurrentTripValue(relativ

e to offset)=668 *>

e to offset)=668 *>



G-a26d988>

G-a26d988>

<

<

* No I2C Devices found *> <* MCP23017 I2C:x

* No I2C Devices found *> <* MCP23017 I2C:x

20 Device not detected *> <* MCP23017 I2C:x21 Dev

20 Device not detected *> <* MCP23017 I2C:x21 Dev

ice not detected *> <* Signal pin config: high a

ice not detected *> <* Signal pin config: high a

c

c

curacy waveform *> <* LCD3:Ready *

curacy waveform *> <* LCD3:Ready *

> <* LCD2:Pow

> <* LCD2:Pow

e

e

r Off *> PPA0 <* LCD3:Free RAM= 5788b *>

r Off *> PPA0 <* LCD3:Free RAM= 5788b *>
```

**Comment by alex-code** [comment]
Could you try the hosted version and see if that works.
https://dcc-ex.com/WebThrottle-EX/

**Reply by fwelto** [reply]
Thanks Alex.  I appreciate the reply.  I got it working.  I noticed I had both the Arduino Serial Monitor and the webthrottleEX open at the same time so I closed both and re-opened webthreottleEX and it started working as expected.  Maybe there was a contention for the serial port?

**Reply by alex-code** [reply]
I think serial ports are exclusive so only one connection at a time. Glad you got it working.
